articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,487 @@
+description: Monitor ASP.NET Core web applications for availability, performance, and usage.
+ms.topic: conceptual
+ms.devlang: csharp
+ms.custom: devx-track-csharp
+ms.date: 08/03/2022
+ms.reviewer: casocha
+---
+
+# Application Insights for ASP.NET Core applications
+
+This article describes how to enable Application Insights for an [ASP.NET Core](https://docs.microsoft.com/aspnet/core) application deployed as an Azure Web App. This implementation utilizes an SDK-based approach, an [auto-instrumentation approach](./codeless-overview.md) is also available.
+
+Application Insights can collect the following telemetry from your ASP.NET Core application:
+
+> [!div class="checklist"]
+> * Requests
+> * Dependencies
+> * Exceptions
+> * Performance counters
+> * Heartbeats
+> * Logs
+
+We'll use an [ASP.NET Core MVC application](https://docs.microsoft.com/aspnet/core/tutorials/first-mvc-app) example that targets `net6.0`. You can apply these instructions to all ASP.NET Core applications. If you're using the [Worker Service](https://docs.microsoft.com/aspnet/core/fundamentals/host/hosted-services#worker-service-template), use the instructions from [here](./worker-service.md).
+
+> [!NOTE]
+> A preview [OpenTelemetry-based .NET offering](./opentelemetry-enable.md?tabs=net) is available. [Learn more](./opentelemetry-overview.md).
+
+[!INCLUDE [azure-monitor-log-analytics-rebrand](../../../includes/azure-monitor-instrumentation-key-deprecation.md)]
+
+## Supported scenarios
+
+The [Application Insights SDK for ASP.NET Core](https://nuget.org/packages/Microsoft.ApplicationInsights.AspNetCore) can monitor your applications no matter where or how they run. If your application is running and has network connectivity to Azure, telemetry can be collected. Application Insights monitoring is supported everywhere .NET Core is supported. Support covers the following scenarios:
+* **Operating system**: Windows, Linux, or Mac
+* **Hosting method**: In process or out of process
+* **Deployment method**: Framework dependent or self-contained
+* **Web server**: IIS (Internet Information Server) or Kestrel
+* **Hosting platform**: The Web Apps feature of Azure App Service, Azure VM, Docker, Azure Kubernetes Service (AKS), and so on
+* **.NET Core version**: All officially [supported .NET Core versions](https://dotnet.microsoft.com/download/dotnet-core) that aren't in preview
+* **IDE**: Visual Studio, Visual Studio Code, or command line
+
+## Prerequisites
+
+If you'd like to follow along with the guidance in this article, certain pre-requisites are needed.
+
+* Visual Studio 2022
+* Visual Studio Workloads: ASP.NET and web development, Data storage and processing, and Azure development
+* .Net 6.0
+* Azure subscription and user account (with the ability to create and delete resources)
+
+## Deploy Azure resources
+
+Please follow the guidance to deploy the sample application from its [GitHub repository.](https://github.com/solliancenet/appinsights-azurecafe).
```

Comment:
@AaronMaxwell @cartersocha - I'll let you decide where to host this sample app. We should not link to any non Microsoft, non Azure github repos.